vic/drivers/shared_all/src/vic_time.c

```diff
@@ -851,7 +853,21 @@ time_delta(dmy_struct        *dmy_current,
             }
         }
         else if (freq == FREQ_NYEARS) {
+            // advance year
             dmy_next.year += n;
+            // re-calculate day_in_year (NOTE: currently will have problem
+            // if dmy_next.year is not leap year but date is Feb 29 !)
```

Comment:
@bartnijssen It is after the fix. This note is for the purpose of reminder for future code improvement. There will only be a problem if a simulation starts on Feb 29 of a leap year, and yearly aggregation/output is used for AGGFREQ or HISTFREQ (which should be rare case of usage...). We can definitely fix this potential problem also, but it requires some more logics. For now I just added an error message below saying that this is a simulation time case that VIC does not support now, if that's fine.